src/components/message/index.tsx

```diff
@@ -276,11 +278,17 @@ export class Message extends React.Component<Properties, State> {
   }
 
   renderBody() {
-    const { message } = this.props;
+    const { message, isHidden } = this.props;
 
     return (
       <div {...cn('block-body')}>
-        {message && <ContentHighlighter message={message} />}
+        {message && (
+          <ContentHighlighter
+            message={message}
+            isHidden={isHidden}
+            onHiddenMessageInfoClick={this.props.onHiddenMessageInfoClick}
+          />
+        )}
```

Comment:
Makes sense. Yeah will merge this one, and do it in a follow up. Originally..i thought to avoid adding a separate component for this because i thought the code for rendering that message..is too small. design question 😅